### ISSUE TITLE
Explore incorporating ESLint JSX a11y rules

### ISSUE THREAD

**Issue by nicklemmon** [SparkPost/matchbox#203]
In order to help promote best practice, trying out this plugin could be helpful:
[ESLint Plugin JSX a11y](https://www.npmjs.com/package/eslint-plugin-jsx-a11y)

In the past, this plugin came with some false positives or rules that weren't actually enforcing things that ultimately had an impact on the end user. Those rules should be explored as well, and potentially disabled.
